CHANGES

```diff
@@ -4,6 +4,10 @@
 
  Changes between 1.1.0e and 1.1.1 [xx XXX xxxx]
 
+  *) Add the z modifier parsing to BIO_printf() et al formatting string,
```

Comment:
IMHO : ... the z format-modifier ...
Text is a bit different from PR title , is it ok ?